src/routes/exception/exception.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as express from 'express';
+import propertyExceptionRouter from './property';
+import missingEntityExceptionRouter from './missing-entity';
+
+const exceptionRouter = express.Router({ mergeParams: true });
+exceptionRouter.use('/property', propertyExceptionRouter);
+exceptionRouter.use('/missing-entity', missingEntityExceptionRouter);
+
+export default exceptionRouter;
```

Comment:
Curious, why a separate folder + file for these routes, separate from the rest of the exception routes at `/routes/exception.ts` ?

Reply:
OK I see ... separation into property router vs missing-entity router

Reply:
just organisation. 

the previous exceptions are now "exceptions/property" and these newly added missing entity exception ones are under "exceptions/missing-entity".

containing all exception logic in a single folder & under a single route declaration in app means we could add/remove/extend functionality easier. Example: what if all exception endpoints needed an extra auth check? We could just add a middleware at the `exception` route instead of in two places.

I have noticed there is a duplicate file here. should only be index and not exception for the module exporting.